manifest.yaml

```diff
@@ -35,24 +23,12 @@ data:
   # Used to authenticate the culler to the hub. This string was generated with `openssl rand -hex 32`.
   auth.jhub-token.cull: 13fdff1305cd883e49223908186a63294922dadb59b5d1122473041f160c4b03
 ---
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: data8-hub-workdir-01
-spec:
-  capacity:
-    storage: 10Gi
-  accessModes:
-    - ReadWriteOnce
-  persistentVolumeReclaimPolicy: Retain
-  gcePersistentDisk:
-    pdName: data8-hub-workdir-01
-    fsType: ext4
----
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: data8-workdir
+  annotations:
+    volume.beta.kubernetes.io/storage-class: single-user-storage
```

Comment:
Do you actually think you could make a new storage class for the hub? The name `single-user-storage` is confusing because the `single-user-storage` storage class is for individual users, not the hub.

In addition, it is likely that we'll want to eventually adjust the storage capacity of the single user containers separately from the hub's storage (eg. if students start geting bigger datasets).